[0m15:23:02.215034 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f419ee87dd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f419f0e6300>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f419efa0b00>]}


============================== 15:23:02.246133 | f650c43a-6864-4b7c-af4d-6e44447510e5 ==============================
[0m15:23:02.246133 [info ] [MainThread]: Running with dbt=1.9.4
[0m15:23:02.248511 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'profiles_dir': '/mnt/c/Users/RihabBahri/OneDrive - VISEO/SNOWFLAKE/projet/snowflake-dbt/demo-dbt/dbt', 'log_path': '/mnt/c/Users/RihabBahri/OneDrive - VISEO/SNOWFLAKE/projet/snowflake-dbt/demo-dbt/dbt/logs', 'fail_fast': 'False', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'static_parser': 'True', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'invocation_command': 'dbt debug', 'send_anonymous_usage_stats': 'True'}
[0m15:23:02.299745 [info ] [MainThread]: dbt version: 1.9.4
[0m15:23:02.302302 [info ] [MainThread]: python version: 3.12.3
[0m15:23:02.304937 [info ] [MainThread]: python path: /home/rihabahri/.local/share/pipx/venvs/dbt-snowflake/bin/python
[0m15:23:02.307691 [info ] [MainThread]: os info: Linux-5.15.167.4-microsoft-standard-WSL2-x86_64-with-glibc2.39
[0m15:23:02.355702 [error] [MainThread]: Encountered an error:
Parsing Error
  Env var required but not provided: 'SF_ACCOUNT'
[0m15:23:02.361980 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_success": false, "command_wall_clock_time": 0.44565234, "process_in_blocks": "56968", "process_kernel_time": 0.138294, "process_mem_max_rss": "96632", "process_out_blocks": "0", "process_user_time": 1.76592}
[0m15:23:02.365525 [debug] [MainThread]: Command `dbt debug` failed at 15:23:02.365242 after 0.45 seconds
[0m15:23:02.368951 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f419ee87dd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f419e9cf470>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f419e94daf0>]}
[0m15:23:02.372107 [debug] [MainThread]: Flushing usage events
[0m15:23:02.731385 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m15:25:47.193455 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fe13e8edee0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fe13ea6f200>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fe13e46d1f0>]}


============================== 15:25:47.202460 | da1f24a1-41d6-454c-a9ad-c2ab3ab1e138 ==============================
[0m15:25:47.202460 [info ] [MainThread]: Running with dbt=1.9.4
[0m15:25:47.204501 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/mnt/c/Users/RihabBahri/OneDrive - VISEO/SNOWFLAKE/projet/snowflake-dbt/demo-dbt/dbt', 'fail_fast': 'False', 'version_check': 'True', 'log_path': '/mnt/c/Users/RihabBahri/OneDrive - VISEO/SNOWFLAKE/projet/snowflake-dbt/demo-dbt/dbt/logs', 'debug': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'log_format': 'default', 'invocation_command': 'dbt run --select raw', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m15:25:47.227401 [error] [MainThread]: Encountered an error:
Parsing Error
  Env var required but not provided: 'SF_ACCOUNT'
[0m15:25:47.231656 [debug] [MainThread]: Resource report: {"command_name": "run", "command_success": false, "command_wall_clock_time": 0.14659809, "process_in_blocks": "63072", "process_kernel_time": 0.164103, "process_mem_max_rss": "94332", "process_out_blocks": "0", "process_user_time": 1.702573}
[0m15:25:47.233862 [debug] [MainThread]: Command `dbt run` failed at 15:25:47.233577 after 0.15 seconds
[0m15:25:47.235544 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fe13e46d1f0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fe13df10290>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fe13df10b90>]}
[0m15:25:47.236958 [debug] [MainThread]: Flushing usage events
[0m15:25:47.660304 [debug] [MainThread]: An error was encountered while trying to flush usage events
